framework/framework.go

```diff
@@ -211,7 +211,7 @@ func runStructuredCloudEvent(w http.ResponseWriter, r *http.Request, body []byte
 	enc.SetEscapeHTML(false)
 	err := enc.Encode(event)
 	if err != nil {
-		writeHTTPErrorResponse(w, http.StatusBadRequest, crashStatus, fmt.Sprintf("Unable to construct event: %s", err.Error()))
+		writeHTTPErrorResponse(w, http.StatusBadRequest, crashStatus, fmt.Sprintf("Unable to construct event %v: %s", event, err.Error()))
```

Comment:
Are we sure printing `event` will format print into something legible?

Reply:
It's a map, so it should, right?

Reply:
So, we'll print:
```
Unable to construct event map[key1:val1 key2:val2 key3:val3 key4:val4 key5:val5 key6:val6 key7:val7]: errorobj
```
I guess that's OK.